src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -152,10 +153,29 @@ function splitFileToChunks(filePath: string, chunkSize: number): Chunk[] {
  * This function will return relevant cross files for the given editor file
  * by referencing open files, imported files and same package files.
  */
-async function getRelevantCrossFiles(editor: vscode.TextEditor, dependencyGraph: DependencyGraph): Promise<string[]> {
-    return getOpenFilesInWindow().filter(file => {
+export async function getRelevantCrossFiles(editor: vscode.TextEditor): Promise<string[]> {
```

Comment:
what is a "cross file"? is this getting "nearby files of the same language"?

```suggestion
export async function getNearbyFiles(editor: vscode.TextEditor): Promise<string[]> {
```

Reply:
yea I think Abhinav named it with `getRelevantCrossFiles` just because of the code name of this feature is "CrossFile Context". And this function aims to fetch supplemental files other than the current file where users trigger CodeWhisperer service (thus we call it cross file). I think the name itself is fine for now, it's better to refactor this highest level function into an object afterward like `CodeWhispererFileFetcher` or `CodeWhispererFileCrawler`  thus we can abstract different strategies under this class.

Reply:
Previously this function was using imported files + files within same package/folder as strategy, and now we try opened files which seems to be more effective in terms of recommendation quality.